.ci/.docker-images.yml

```diff
@@ -0,0 +1,336 @@
+---
+# Dockerfile build config for .github/workflows/publish-docker-images.yml
+# Schema: .ci/.docker-images.schema.json
+# See docs/INTERNAL_DOCKER_IMAGES.md for further information.
+
+templates:
+
+  - &apm-agent-library-docker-images
+    repository: "elastic/apm-pipeline-library"
+    working_directory: ".ci/docker"
+
+  - &opbeans-docker-images
+    tag: "daily"
+
+  - &apm-agent-python-docker-images
+    name: "apm-agent-python"
+    repository: "elastic/apm-agent-python"
+    working_directory: "tests"
+
+  - &apm-agent-nodejs-docker-images
+    name: "apm-agent-nodejs"
+    repository: "elastic/apm-agent-nodejs"
+    working_directory: ".ci/docker/node-container"
+
+  - &apm-agent-ruby-docker-images
+    name: "apm-agent-ruby"
+    repository: "elastic/apm-agent-ruby"
+    working_directory: "spec"
+
+  - &apm-agent-rum-docker-images
+    repository: "elastic/apm-agent-rum-js"
+
+  - &observability-dev-docker-images
+    repository: "elastic/observability-dev"
+
+images:
+
+  # observability-test-environments Docker Images
+
+  - name: "functional-opbeans"
+    repository: "elastic/observability-test-environments"
+    working_directory: "tests"
+    build_script: "docker build --force-rm -t ${REGISTRY}/${PREFIX}/${NAME}:${TAG} ${NAME}"
+    push_script: "docker push ${REGISTRY}/${PREFIX}/${NAME}:${TAG}"
+
+## This is also failing in Jenkins
+##  https://apm-ci.elastic.co/job/apm-shared/job/docker-images/job/oracle-instant-client/
+#  - name: "azure-vm-tools"
+#    repository: "elastic/azure-vm-extension"
+#    working_directory: ".ci/docker/azure-vm-tools"
+
+  # observability-robots Docker Images
+
+  - name: "picklesdoc"
+    repository: "elastic/observability-robots"
+    build_script: "make build"
+    push_script: "make push"
+    working_directory: "apps/pickles"
+
+## This is also failing on Jenkins
+## https://apm-ci.elastic.co/job/apm-shared/job/docker-images/job/test-plans/
+#  - name: "test-plans"
```

Comment:
This seems to be still in use.

https://github.com/search?q=org%3Aelastic+test-plans&type=Code

Reply:
Will be handled in https://github.com/elastic/apm-pipeline-library/issues/1992